specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.0-preview.1/TextAnalytics.json

```diff
@@ -77,10 +77,22 @@
               "$ref": "#/definitions/EntitiesResult"
             }
           },
```

Comment:
nit: We will be deprecating v3.0-preview.1 on 9/4 so we can probably get rid of this spec completely.

Reply:
I will handle this in a separate PR if its not an issue